### PR TITLE
CRM-20418 - Not able to select Tokens from dropdown on Message Template

### PR DESCRIPTION
**Pull request by jitendrapurohit**
* [CRM-20418: Not able to select Tokens from dropdown on Message Template](https://issues.civicrm.org/jira/browse/CRM-20418)

### CODE REVIEW

File: templates/CRM/Admin/Form/MessageTemplates.tpl [hunk 1]
```diff
@@ -84,7 +84,7 @@
         <tr>
   </table>
 
-      <div id="msg_html" class="crm-accordion-wrapper crm-html_email-accordion ">
```
Comment (jitendrapurohit) [inline templates/CRM/Admin/Form/MessageTemplates.tpl]: Two ids with the same text were used in the form(div and textarea input), Hence `$('#msg_html').text()` returned an empty text.

**Comment by jitendrapurohit** [comment]
jenkins, test this please

**Comment by alifrumin** [comment]
jenkins, test this please

**Comment by highfalutin** [comment]
reviewing

**Pull request closed without merging**